post_installation_configuration/configuring-private-cluster.adoc

```diff
@@ -17,3 +17,11 @@ include::modules/private-clusters-setting-ingress-private.adoc[leveloffset=+1]
 include::modules/private-clusters-setting-api-private.adoc[leveloffset=+1]
 
 include::modules/nw-ingresscontroller-change-internal.adoc[leveloffset=+2]
+
+include::modules/registry-configuring-private-storage-endpoint-azure.adoc[leveloffset=+1]
+
+include::modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc[leveloffset=+2]
+
+include::modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc[leveloffset=+2]
+
+include::modules/disabling-redirect-private-storage-endpoint-azure.adoc[leveloffset=+3]
```

Comment:
Just checking if this is intentionally at a level 3 indentation? It doesn't show in the TOC on the right then

Reply:
It was intentional, but let me reconsider making it level 2!